### ISSUE TITLE
Per-group identities / display names

### ISSUE THREAD

**Issue by madduck** [loomio/loomio#6329]
With Loomio's Web UI giving me one-stop access to all the different groups/communities using Loomio, of which I am part, it may be such that one has difficulty choosing the single profile display name and/or photo. For instance, I am historically "madduck" around open-source software, but in other contexts, I might want to instead have my full name, or even include a title here or there.

It would be amazing if Loomio allowed me to override my identity based on context, so that I could have a default identity, but change it depending on whom I'm interacting with.

Thank you for your consideration!

**Comment by robguthrie** [comment]
Yes! It's in the plans. We already have a title attribute on the membership record, so it makes sense to include this stuff on membership in the future.

**Comment by Meldane** [comment]
Oh that's great ! Because I ended up with 2 accounts in order to be able to display 2 different profiles depending on context...
If that's the plan, would it also be possible to choose with group I want to show publicly I am part of and with one I would rather not ?
